src/assets/githubAction/githubActionBeginner.md

```diff
@@ -0,0 +1,130 @@
+---
+title: GitHub Actions - CI & CD
```

Comment:
```suggestion
title: GitHub Actions - CI & CD (For Beginners)
```
I think I have to change the topics since its for beginner
Or 
It will be ok 
Or 
need suggestion

Reply:
That's fine. Maybe ask your colleagues for suggestions too :)

You could also say something like "Introduction to GitHub Actions - CI & CD"

And here are some ideas that an AI based title generator tool gave me :)
1. "Mastering GitHub Actions: A Beginner's Guide to CI & CD"
2. "Unleashing the Power of GitHub Actions: A Step-by-Step CI & CD Tutorial for Beginners"
3. "Demystifying GitHub Actions: The Ultimate CI & CD Tool for Beginners"
4. "Boost Your Development Workflow with GitHub Actions: A Beginner's Introduction to CI & CD"
5. "GitHub Actions Made Easy: A Beginner's Guide to Implementing CI & CD in Your Projects"

Reply:
AI made much longer titles than I thought of ;)

Reply:
I think AI can be useful to get more ideas, but some of these AI generated titles might not match with the content of your blog post. Also decide on a title that you feel comfortable with and that uses words you would also be using.